pkgs/leak_tracker_flutter_testing/lib/src/matchers.dart

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) 2023, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:leak_tracker_testing/leak_tracker_testing.dart';
+
+/// Checks if the object dispatches events to `MemoryAllocations.instance`.
+const Matcher dispatchesMemoryEvents = _DispatchesMemoryEvents();
```

Comment:
Most matchers that don't take arguments use the verbiage `is$Something`. Would something like `isDisposed` work here?

Reply:
After some thinking redesigned to have arguments. Seems to be cleaner.